test/functional/mining_prioritisetransaction.py

```diff
@@ -131,7 +134,11 @@ def run_test(self):
         self.relayfee = self.nodes[0].getnetworkinfo()['relayfee']
 
         utxo_count = 90
-        utxos = create_confirmed_utxos(self, self.relayfee, self.nodes[0], utxo_count)
+        # TODO: use helper create_confirmed_utxos as soon as it uses MiniWallet
```

Comment:
I am not a fan of putting TODO in the source code. If (for any reason) they become stale, they will cause more damage/confusion than helping.

It would be better as an issue, so that it can be tracked easier as the code evolves. Also it can be discussed easier and discarded trivially if needed without touching the code.

Moreover, it seems that `send_large_txs` can be replaced by `create_lots_of_big_transactions`?
Same for `mine_large_block`?

Reply:
> It would be better as an issue, so that it can be tracked easier as the code evolves. Also it can be discussed easier and discarded trivially if needed without touching the code.

I see your point, though I think creating an issue for merely refactoring three lines of code is a bit like taking a sledgehammer to crack a nut (and one also has to think about opening it after the PR is merged which "triggers" the TODO). Removed the TODO for now -- replacing `create_confirmed_utxos` (and feature_dbcrash.py) with MiniWallet is one of the next things I'll work on anyways and in the course of that I'll check where it can dedup code.

> Moreover, it seems that send_large_txs can be replaced by create_lots_of_big_transactions?
Same for mine_large_block?

Good idea, I added a second refactoring commit doing exactly that. For that to work, I had to generalize `create_lots_of_big_transactions` a bit, in particular the possibility to not pass in any UTXOs (MiniWallet picks one of it's internal UTXOs then in this case). Also it validates the fee now, like it's done in `send_large_txs`, which I think makes a lot of sense.